docs/en/best_practices/bucketing.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 30
+---
+
+# Bucketing
+
+A concise field guide to choosing between Hash Bucketing and Random Bucketing in StarRocks, including their mechanics, trade‑offs, and recommended use cases.
+
+---
+## Quick‑Look Comparison
+
+| Aspect | Hash Bucketing | Random Bucketing |
+| ------ | -------------- | ---------------- |
+| Key declaration | Required HASH(col1, …) | None – rows assigned round‑robin |
```

Comment:
Add an example here

Reply:
Can you make the change please @murphyatwork I do not know what you mean.